common/src/main/resources/assets/voicechat/lang/de_de.json

```diff
@@ -189,5 +197,7 @@
   "cloth_config.voicechat.config.group_hud_icon_scale": "Gruppenoberflächen Symbol Skalierung",
   "cloth_config.voicechat.config.group_player_icon_pos_x": "Gruppenoberflächen Symbol X position",
   "cloth_config.voicechat.config.group_player_icon_pos_y": "Gruppenoberflächen Symbol Y position",
-  "cloth_config.voicechat.config.show_own_group_icon": "Eigenes Gruppenoberflächen Symbol anzeigen"
+  "cloth_config.voicechat.config.show_own_group_icon": "Eigenes Gruppenoberflächen Symbol anzeigen",
+  "cloth_config.voicechat.config.display_nametags": "Namensschilder anzeigen",
+  "cloth_config.voicechat.config.on_talk_nametags": "Namensschilder während Gespräche"
```

Comment:
The translations seem to be made with a translator. Please just don't do any translations for languages you don't know.